KalturaMediaServer/src/com/kaltura/media/server/managers/KalturaCuePointsManager.java

```diff
@@ -118,8 +120,16 @@ public void load(final List<String> entryIds, boolean periodic) {
 						}
 						
 						// create sync-point 30 seconds before the trigger time
-						date.setTime((cuePoint.triggeredAt / 1000) - 30000);
+						long scheduledTime = (cuePoint.triggeredAt / 1000) - 30000;
```

Comment:
This was inserted due to cases where the triggered at was small and created a situation that the Timer failed, consider the fact that the user can set values through API calls